### DISCUSSION TITLE
MSI Katana GF76 11UEK - Rainbow Keyboard

### DISCUSSION THREAD

**Discussion by Claod44** [ixjf/MSIRGB#223] [General]
Hi,
With this project is there a way to set a color gradient for the whole keyboard : The 1-part in Green, The 2-part in blue etc ? Please
And if it's not the case, is this possible in the future ?

**Comment by ixjf** [comment]
Nope, MSIRGB only supports changing motherboard LEDs and I have no plans to add any further support. Sorry.

**Reply by Claod44** [reply]
Hello, 
Okey, thank you for your answer.  I was asking this because Mystic light is not changing the default (blue) color on each cold boot. 
The rainbow effect will not be possible because I think LEDs are not addressable on my motherboard, sa bad.
On the description of MSIRGB my motherboard is not listed, can you confirm the Micro-star MS-17L1 is not supported on your project please ?

**Reply by ixjf** [reply]
It's not.
